server/package.json

```diff
@@ -18,6 +18,7 @@
     "db-migrate-plugin-typescript": "2.0.0",
     "event-stream": "3.3.5",
     "express": "4.17.2",
+    "express-json-validator-middleware": "^2.2.1",
```

Comment:
```suggestion
    "express-json-validator-middleware": "2.2.1",
```

Reply:
We don't need to lock in dependencies do we?

Reply:
Renovate is configured to lock dependancies and automerge anything but major (breaking) changes. So this just saves renovate from opening a PR to pin this.